README.md

```diff
@@ -1,4 +1,6 @@
-# flyte client
+# flyte-client
+
+![Build Status](https://travis-ci.org/HotelsDotCom/flyte-bamboo.svg?branch=master)
```

Comment:
everyone else has this above the title I think?

Reply:
https://github.com/kubernetes/kubernetes ;)